gwt-regexp/pom.xml

```diff
@@ -65,39 +67,26 @@
             <version>${elemental2.version}</version>
             <classifier>sources</classifier>
         </dependency>
+        <dependency>
```

Comment:
Is this actually needed? Are there any tests left in this module?

Reply:
I moved plain JUnit tests to the module itself, it was a separate module

Reply:
This should be discussed, as we don't wanted any tests inside the module. I prefer having a separate for test.

Reply:
I always prefer tests inside the module as they should be one component. Separating j2cl and gwt tests is a must for now but eventually they must be done in a better way. I don't see an added value for separating plain Junit tests in different module.

Reply:
Generally agreed, but for now this is the cleanest way while gwt2 still needs GwtTestCase - that won't change for quite some time.

Reply:
Ok, confirmed that there are tests in this module, but JVM tests only, which we do still want.